dist_run.py

```diff
@@ -182,7 +195,8 @@ def main():
 
     # Distribute model on TP mesh
     model.distribute(tp_mesh)
-    logger.info(f"Model: {model}")
+    if rank == 0:
```

Comment:
nit: already filtering based on ranks in the dist_run.sh launcher rather than having express rank based logging in the code.

Reply:
Yeah, it is a hard choice.. I don't know if regular users use filter (I never remember that option's name) and want to have a minimal command line work.